src/rgen/Microsoft.Macios.Bindings.Analyzer.Sample/Examples.cs

```diff
@@ -30,4 +32,3 @@ public enum Test {
 	}
 
 }
-
```

Comment:
nit but was this intended lol

Reply:
It was, all other files don't have it, and the script that I used cleaned it.